### PR TITLE
[change] scope no-onchange rule to select menu elements only

### PR DESCRIPTION
**Pull request by octatone**
fixes https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/55

### CODE REVIEW

File: docs/rules/no-onchange.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+Enforce usage of `onBlur` over/in parallel with `onChange` on select menu elements for accessibility. `onBlur` **should** be used instead of `onChange`, unless absolutely necessary and it causes no negative consequences for keyboard only or screen reader users. `onBlur` is a more declarative action by the user: for instance in a dropdown, using the arrow keys to toggle between options will trigger the `onChange` event in some browsers. Regardless, when a change of context results from an `onBlur` event or an `onChange` event, the user should be notified of the change unless it occurs below the currently focused element.
```
Comment (evcohen) [inline docs/rules/no-onchange.md]: Can you update the examples at the bottom too?

File: tests/src/rules/no-onchange.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * @fileoverview Enforce usage of onBlur over onChange for accessibility.
+ * @fileoverview Enforce usage of onBlur over onChange on select menus for accessibility.
```
Comment (evcohen) [inline tests/src/rules/no-onchange.js]: update `README.md` please with updated summary

File: docs/rules/no-onchange.md (hunk 1 above)
Comment (octatone) [inline docs/rules/no-onchange.md]: done ✔️

File: tests/src/rules/no-onchange.js (hunk 2 above)
Comment (octatone) [inline tests/src/rules/no-onchange.js]: done ✔️

**Comment by octatone** [comment]
squashed

**Comment by cristian-sima** [comment]
Anything on this ?

**Comment by evcohen** [comment]
@octatone can you rebase, will get this merge in today. will probably be a major bump, so wanted to wait to see if we could bundle other changes in with it...screw it 😛

**Comment by octatone** [comment]
Sure thing, rebase incoming ...

**Comment by octatone** [comment]
rebased

**Comment by evcohen** [comment]
[v2.0.0](https://github.com/evcohen/eslint-plugin-jsx-a11y/releases/tag/2.0.0) published, thank you!

**Pull request merged**